packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -771,6 +771,20 @@ export interface CapacityOptions extends autoscaling.CommonAutoScalingGroupProps
    * @default - none
    */
   readonly bootstrapOptions?: BootstrapOptions;
+
+  /**
+   * Specifies how block devices are exposed to the instance. You can specify virtual devices and EBS volumes.
+   *
+   * Each instance that is launched has an associated root device volume,
+   * either an Amazon EBS volume or an instance store volume.
+   * You can use block device mappings to specify additional EBS volumes or
+   * instance store volumes to attach to an instance when it is launched.
+   *
+   * @see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/block-device-mapping-concepts.html
+   *
+   * @default - Uses the block device mapping of the AMI
+   */
+  readonly blockDevices?: autoscaling.BlockDevice[];
```

Comment:
Wouldn't it make more sense to add this to `autoscaling.CommonAutoScalingGroupProps`?

Reply:
yes, let me add to autoscaling.CommonAutoScalingGroupProps